.github/CODEOWNERS

```diff
@@ -188,6 +188,7 @@
 
 # PRLabel: %ML-Jobs
 /sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/                         @DouglasXiaoMS @TonyJ1 @wangchao1230
+/sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job*                         @DouglasXiaoMS @TonyJ1 @wangchao1230
```

Comment:
Do we need both lines or can we just have 191?

Reply:
You are right, we can have one. I confirmed it [empirically](https://github.com/konrad-jamrozik/scratchpad/blob/63bd5cfdfdf56ec1651e3df64155d8b7ec187036/.github/CODEOWNERS#L2).